generator/integration_tests/golden/CMakeLists.txt

```diff
@@ -114,20 +114,20 @@ function (google_cloud_cpp_generator_define_golden_tests)
 
     set(google_cloud_cpp_generator_golden_tests
         # cmake-format: sort
-        golden_client_test.cc
-        golden_connection_test.cc
-        golden_idempotency_policy_test.cc
-        golden_logging_decorator_test.cc
-        golden_metadata_decorator_test.cc
-        golden_stub_factory_test.cc
-        golden_stub_test.cc
-        iam_credentials_client_test.cc
-        iam_credentials_connection_test.cc
-        iam_credentials_idempotency_policy_test.cc
-        iam_credentials_logging_decorator_test.cc
-        iam_credentials_metadata_decorator_test.cc
-        iam_credentials_stub_factory_test.cc
-        iam_credentials_stub_test.cc)
+        unit_tests/golden_client_test.cc
```

Comment:
If this directory could simply be called `tests`, I think that would be cleaner and more common.